sklearn/cluster/k_means_.py

```diff
@@ -40,8 +40,8 @@
 ###############################################################################
 # Initialization heuristic
 
-
-def _k_init(X, n_clusters, x_squared_norms, random_state, n_local_trials=None):
+def _k_init(X, n_clusters, x_squared_norms, random_state, n_local_trials=None,
+            check_inputs=True):
```

Comment:
My question was more: is it called with both "True" and "False" in the current code?